datalab/datalab_session/data_operations/rgb_stack.py

```diff
@@ -141,21 +131,20 @@ def operate(self):
             dir=settings.TEMP_FITS_DIR
         ) as (tif_path, large_jpg_path, small_jpg_path):
         
-            create_tif(aligned_images, tif_path, color=True, zmin=zmin_list, zmax=zmax_list)
-            create_jpgs(aligned_images, large_jpg_path, small_jpg_path, color=True, zmin=zmin_list, zmax=zmax_list)
-            
-            stacked_ndarray = self._create_3d_array(input_handlers)
-            rgb_comment = f'Datalab RGB Stack on files {", ".join(input["basename"] for input in rgb_inputs)}'
-
-            output = FITSOutputHandler(
-                self.cache_key, 
-                stacked_ndarray, 
-                rgb_comment
-            ).create_and_save_data_products(
-                large_jpg_path=large_jpg_path, 
-                small_jpg_path=small_jpg_path,
-                tif_path=tif_path
-            )
+            try:
+                create_tif(aligned_images, tif_path, color=True, zmin=zmin_list, zmax=zmax_list)
+                create_jpgs(aligned_images, large_jpg_path, small_jpg_path, color=True, zmin=zmin_list, zmax=zmax_list)
+            except Exception as ex:
+                # Catches exceptions in the fits2image methods to report back to frontend
+                raise ClientAlertException(ex)
```

Comment:
The fits2image library throws some exceptions if the input images shapes don't match well - This try/except will catch those exceptions and re-raise them so the UI can show the error to the user instead of the generic error.